mbofui/app/js/controllers.js

```diff
@@ -204,10 +204,10 @@ ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter
     // use a promise factory to do request
     Bof.GetBofs(bofsUrl).then(function(eventsList) {
       Bof.GetIntentions($rootScope.urls.intentions + '?username=self').then(function(intentionsList) {
-        $scope.intentions=intentionsList;
+        $scope.intentions=intentionsList.data.results;
```

Comment:
What kind of data is held in the intentions list data results? What is the structure? Event, User, Response, etc.?